### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by gempa-jabe**
@gempa-dirk updated the documentation and module descriptions. Units were added and the documentation changes were ported from SeisComP3. Please check again if everything is correct.

### CODE REVIEW

**Comment by luca-s** [comment]
Sorry, I totally missed the notification for this PR. I'll review it right away

File: apps/scenvelope/config/scenvelope.cfg [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
-# This parameter defines the length of the buffer (in seconds) that is used to 
-# compute a real-time average for baseline correction. 
+# This parameter defines the length of the buffer (in seconds) that is used to
+# compute a real-time average for baseline correction.
 envelope.baselineCorrectionBuffer=60
 
-# This is for internal testing purposes only. 'true' will cause the 
-# SeisComP filter routines to be used. If 'false' the filter routines 
+# This is for internal testing purposes only. 'true' will cause the
+# SeisComp3 filter routines to be used. If 'false' the filter routines
```
Comment (luca-s) [inline apps/scenvelope/config/scenvelope.cfg:36]: Strange, SeisComP has been changed to SeisComp3. Shouldn't be the opposite going further?

File: apps/scenvelope/descriptions/scenvelope.rst [hunk 2]
```diff
@@ -1,31 +1,62 @@
 Part of the :ref:`VS` package.
 
-scenvelope is part of a new SeisComP implementation of the
-`Virtual Seismologist <http://www.seismo.ethz.ch/research/vs>`_ (VS) Earthquake
+scenvelope is part of a new SeisComp3 implementation of the
+`Virtual Seismologist`_ (VS) Earthquake
```
Comment (luca-s) [inline apps/scenvelope/descriptions/scenvelope.rst:4]: SeisComP3 but it should be SeisComP

File: apps/scenvelope/descriptions/scenvelope.rst [hunk 3]
```diff
@@ -1,31 +1,62 @@
...
 Early Warning algorithm (Cua, 2005; Cua and Heaton, 2007) released
-under the `'SED Public License for SeisComP Contributions' 
-<http://www.seismo.ethz.ch/static/seiscomp_contrib/license.txt>`_. It generates
+under the `SED Public License for SeisComP Contributions`_. It generates
 real-time envelope values for horizontal and vertical acceleration, velocity and
```
Comment (luca-s) [inline apps/scenvelope/descriptions/scenvelope.rst:7]: "under the terms of the GNU Affero General Public License" but I guess this is on me to change that

File: apps/scenvelope/descriptions/scenvelope.xml [hunk 4]
```diff
@@ -43,7 +43,7 @@
 				<parameter name="useSC3Filter" type="boolean" default="false">
 					<description>
 					This is for internal testing purposes only. true will cause the
-					SeisComP filter routines to be used. If false the filter routines
+					SeisComp3 filter routines to be used. If false the filter routines
```
Comment (luca-s) [inline apps/scenvelope/descriptions/scenvelope.xml]: Revert back to SeisComp

File: apps/scvsmag/descriptions/VS.rst [hunk 5]
```diff
@@ -1,11 +1,11 @@
-The Virtual Seismologist in SeisComP (VS(SC3)) provides near instantaneous
-estimates of earthquake magnitude as soon as SeisComP origins are available. With a
-well-configured SeisComP system running on a dense network, magnitudes for
-local events can be available within 10-20 s of origin time. VS(SC3) can be a key
+The  `Virtual Seismologist`_ in SeisComP (VS(SC)) provides near instantaneous
+estimates of earthquake magnitude as soon as SeisComp3 origins are available. With a
+well-configured SeisComP3 system running on a dense network, magnitudes for
+local events can be available within 10-20 s of origin time. VS(SC) can be a key
 component of an earthquake early warning system, and can be used to provide
```
Comment (luca-s) [inline apps/scvsmag/descriptions/VS.rst:5]: Revert back SeisComP3 to SeisComP

File: apps/scvsmag/descriptions/VS.rst [hunk 6]
```diff
@@ -145,9 +147,9 @@ See :ref:`scvsmag` on how the VS specific solution quality is computed.
 VS License
 ----------
 
-The SeisComP VS modules are free and open source, and are part of the SeisComP
-distribution from Seattle v2013.200. They are distributed under the `'SED Public
-License for SeisComP Contributions' <http://www.seismo.ethz.ch/static/seiscomp_contrib/license.txt>`_.
+The SeisComp3 VS modules are free and open source, and are part of the SeisComp3
+distribution from Seattle v2013.200. They are distributed under the `SED Public
+License for SeisComP Contributions`_.
```
Comment (luca-s) [inline apps/scvsmag/descriptions/VS.rst:153]: "under the terms of the GNU Affero General Public License" but I guess this is on me to change that

File: apps/scvsmag/descriptions/scvsmaglog.rst [hunk 7]
```diff
@@ -1,25 +1,22 @@
...
 *scvsmaglog* is part of a new SeisComP implementation of the
-`Virtual Seismologist <http://www.seismo.ethz.ch/research/vs>`_
+`Virtual Seismologist`_
 (VS) Earthquake Early Warning algorithm (Cua, 2005; Cua and Heaton, 2007) released
-under the `'SED Public License for SeisComP Contributions'
-<http://www.seismo.ethz.ch/static/seiscomp_contrib/license.txt>`_. It requires 
-the Python package `dateutil <https://pypi.python.org/pypi/python-dateutil>`_ to
-be installed.
+under the `SED Public License for SeisComP Contributions`_. It requires
+the Python package `dateutil`_ to be installed.
```
Comment (luca-s) [inline apps/scvsmag/descriptions/scvsmaglog.rst:8]: "under the terms of the GNU Affero General Public License" but I guess this is on me to change that

**Review by luca-s — COMMENTED** [review]
This seems ok except many of the occurences of SeisComP that are now SeisComP3. shouldn't that be the opposite going further?

File: apps/scenvelope/config/scenvelope.cfg (hunk 1 above)
Comment (gempa-jabe) [reply apps/scenvelope/config/scenvelope.cfg:36]: WIll be fixed. I just took that from the SC3 documentation changes.

File: apps/scenvelope/descriptions/scenvelope.rst (hunk 3 above)
Comment (gempa-jabe) [reply apps/scenvelope/descriptions/scenvelope.rst:7]: I took that from the changes that @FMassin pushed to SC3 I think.

**Comment by gempa-jabe** [comment]
I will address your remarks.

**Comment by luca-s** [comment]
Thanks!

File: apps/scenvelope/descriptions/scenvelope.rst (hunk 3 above)
Comment (FMassin) [reply apps/scenvelope/descriptions/scenvelope.rst:7]: I guess it has been reintroduced when I pushed some precisions in the doc https://github.com/SeisComP3/seiscomp3/pull/265... sorry I did not mean to interfere.

**Pull request merged**